juriscraper/opinions/united_states/state/neb.py

```diff
@@ -8,36 +9,65 @@ def __init__(self, *args, **kwargs):
         self.url = (
             "https://supremecourt.nebraska.gov/courts/supreme-court/opinions"
         )
-        self.status = "Published"
+
+    def _return_response_text_object(self):
+        """Remove faulty URLs from HTML
+
+        Override _return_response_text_object in Abstract Site to remove any
+        url to [site:url-brief] which causes lxml/ipaddress to explode.
+
+        This bug only exists in 3.11 and 3.12
+
+        <a href="https://[site:url-brief]/node/19063" ... >District Map</a>
+
+        :return: The cleaned html tree of the site
+        """
+        if self.request["response"]:
+            payload = self.request["response"].text
+            text = self._clean_text(payload)
+            html_tree = self._make_html_tree(text)
+
+            for tag in html_tree.xpath(
+                "//a[contains(@href, 'site:url-brief')]"
+            ):
+                parent = tag.getparent()
+                if parent is not None:
+                    parent.remove(tag)
+
+            if hasattr(html_tree, "rewrite_links"):
+                html_tree.rewrite_links(
+                    fix_links_in_lxml_tree, base_href=self.request["url"]
+                )
+            return html_tree
 
     def _process_html(self):
-        path = {
-            "group": '//div[contains(@class, "view-grouping")]',
-            "date": './/span[contains(@class, "date-display-single")]',
-            "row": ".//tbody/tr",
-        }
-        for group in self.html.xpath(path["group"]):
-            date_element = group.xpath(path["date"])
-            if not date_element:
+        for table in self.html.xpath(".//table"):
+            date_tags = table.xpath("preceding::time[1]/text()")
+            if not date_tags:
                 continue
-            date = date_element[0].text_content()
-            for row in group.xpath(path["row"]):
-                url = row.xpath(".//td[3]//a/@href")
-
-                # skip rows without url in third cell
-                if not url:
-                    continue
+            date = date_tags[0]
 
-                docket = row.xpath(".//td[1]")[0].text_content().strip()
-                if docket == "A-XX-XXXX":
+            for row in table.xpath(".//tr/td/.."):
```

Comment:
It seems like you are testing for existance of a td? Then the precise xpath is  `".//tr[td]"`